api/swagger.json

```diff
@@ -0,0 +1,1304 @@
+            "schema": {
+              "type": "integer",
+              "format": "int64"
+            }
+          },
+          {
+            "name": "eventId",
+            "in": "path",
+            "description": "ID of event to return",
+            "required": true,
+            "schema": {
+              "type": "integer",
+              "format": "int64"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Event"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Not authorized"
+          },
+          "400": {
+            "description": "Invalid event"
+          },
+          "404": {
+            "description": "Event/Chapter not found"
+          },
+          "default": {
+            "description": "unexpected error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "chapter"
+        ],
+        "summary": "Delete an event by id",
+        "description": "",
+        "operationId": "deleteEventById",
+        "parameters": [
+          {
+            "name": "chapterId",
+            "in": "path",
+            "description": "ID of chapter to return",
+            "required": true,
+            "schema": {
+              "type": "integer",
+              "format": "int64"
+            }
+          },
+          {
+            "name": "eventId",
+            "in": "path",
+            "description": "ID of event to return",
+            "required": true,
+            "schema": {
+              "type": "integer",
+              "format": "int64"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Event"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Not authorized"
+          },
+          "400": {
+            "description": "Invalid event"
+          },
+          "404": {
+            "description": "Event/Chapter not found"
+          },
+          "default": {
+            "description": "unexpected error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/chapters/{chapterId}/events/{eventId}/sponsors": {
```

Comment:
Per my comment above, we could drop the first two URL segments and not lose anything of value here.

Reply:
Again, I think the method/route name should indicate the expectation of whether one or multiple entities are being retrieved or sent.